K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -458,7 +458,7 @@ private static TopHitsAggregate GetTopHitsAggregate(string columnName, string[]
             topHitsAggregate.Hits.SetTotal(0);
 
             var rowValue = row[columnName];
-            if (rowValue.GetType() != typeof(System.DBNull))
+            if (rowValue.GetType() != typeof(DBNull))
```

Comment:
If we are touching this line anyway (and in the next one too)
```suggestion
            if (rowValue is not DBNull)
```